table/block_based_table_builder.cc

```diff
@@ -871,6 +871,35 @@ void BlockBasedTableBuilder::WriteRangeDelBlock(
   }
 }
 
+void BlockBasedTableBuilder::WriteFooter(BlockHandle& metaindex_block_handle,
+                                         BlockHandle& index_block_handle) {
+  Rep* r = rep_;
```

Comment:
I think using `rep_` is more search-friendly.

Reply:
I think I agree, but this is mostly based on how the current code is structured. I didn't modify much of the internals during the  refactoring as its easy to introduce new bugs.